### PR TITLE
2 0 refacto rights voters

### PR DESCRIPTION
**Pull request by ngilain**
[OO-FEATURE] Refacto the right layer
https://github.com/open-orchestra/open-orchestra/pull/1087
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1997
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/322
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/240
https://github.com/open-orchestra/open-orchestra-model-interface/pull/256
https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/156
https://github.com/open-orchestra/open-orchestra-base-api-bundle/pull/113

### CODE REVIEW

File: MediaAdmin/Perimeter/Strategy/MediaFolderPerimeterStrategy.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            foreach ($perimeter->getItems() as $path) {
+                if (0 === strpos($path, $item)) {
+                    return true;
+                }
+            }
+        }
+
+        return false;
+    }
+}
```
Comment (alavieille) [inline MediaAdmin/Perimeter/Strategy/MediaFolderPerimeterStrategy.php]: eof

File: MediaAdmin/Tests/Security/Authorization/Voter/AbstractVoterTest.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    protected function createPhakeMedia()
+    {
+        $folder = $this->createPhakeMediaFolder();
+
+        $media = Phake::mock('OpenOrchestra\Media\Model\MediaInterface');
+        Phake::when($media)->getMediaFolder()->thenReturn($folder);
+
+        return $media;
+    }
+}
```
Comment (alavieille) [inline MediaAdmin/Tests/Security/Authorization/Voter/AbstractVoterTest.php]: erof

**Review by alavieille — COMMENTED** [review]
else eof :+1:

**Comment by ngilain** [comment]
@alavieille : update done

**Comment by alavieille** [comment]
:+1:

**Pull request merged**